pkgs/top-level/all-packages.nix

```diff
@@ -17103,6 +17103,8 @@ with pkgs;
 
   gxmessage = callPackage ../applications/misc/gxmessage { };
 
+  gxplugins.lv2 = callPackage ../applications/audio/gxplugins.lv2 { };
```

Comment:
You cannot have a dot in an attribute name because nix will interpret that as the attibute `lv2` of the set `gxplugins`, which doesn't make sense here (that's why ofborg says zero builds). Please rename consistently to `gxplugins-lv2` (or another acceptable name).

Reply:
Thanks for the feedback.
I'm happy to make this change, but I'd like to understand your comment.

>ofborg says zero builds

Where does it say that?

Should I also rename:
- sisco.lv2 
- eteroj.lv2 
- ir.lv2
?

Reply:
Having a top-level set `gxplugins` only makes sense if you want to group multiple derivations that logically belong together under a common name, like `gxplugins.lv2`, `gxplugins.foo`, `gxplugins.bar`, and you put them in a common `gxplugins` subdirectory below `pkgs`.  This doesn't seem to be the case here. Some examples in nixpkgs include `xorg`, `gnome3`, `vimPlugins`, `gitAndTools`, `xfce`. 

> Where does it say that?

Usually there's a "Details" link next to the ofborg eval check to a list of  all  (re)builds caused by the PR. This is missing here because ofborg didn't pick up the new package. But that's not an issue by itself, rather the reason why I noticed your're creating a top-level set. Nevermind.

> Should I also rename:   sisco.lv2 ...

If these are similar cases, yes. If you think they logically belong together because they're all lv2 plugins (your call), it might make sense to group all these in a top-level `lv2-plugins` set:  `lv2-plugins.sisco`, `lv2-plugins.eteroj`, `lv2-plugins.gxplugins` ...  If you want to do that, put them under an a common subdirectory `pkgs/audio/lv2-plugins` and use `recurseIntoAttrs` in `top-level/all-packages.nix`, see the other examples mentioned.

Reply:
@xeji Thanks, done.